docs/index.rst

```diff
@@ -17,6 +17,8 @@ The WaterTAP development team is composed of researchers from:
 * Lawrence Berkeley National Laboratory
 * National Renewable Energy Laboratory
 * Oak Ridge National Laboratory
+* SLAC National Accelerator Laboratory
+* Stanford University
```

Comment:
Adding SLAC to the list here makes sense since it's a national lab, but do we really want to list out all of our university collaborators? If we include Stanford University, we should probably add a dozen more universities as well